### DISCUSSION TITLE
Welcome, Júlio Cézar!

### DISCUSSION THREAD

**Discussion by juletopi** [ParanoidUser/yolo#522] [Subscribe]
_Roses are red, violets are blue...
I post here my request...
So I can get the collaborator permission's too!_ 

😊

**Comment by ParanoidUser** [comment]
Ahoy @juletopi, welcome aboard!

The invitation to join the collaborator group has been sent to you. Thanks for your interest in this project!
Feel free to post your own questions to the community, or share your path to earning other badges. 😊 Let me know if you have any questions.

Cheers
